README.md

```diff
@@ -1,6 +1,6 @@
-# Starlite sphinx theme
+# Litestar sphinx theme
 
-A Sphinx theme for the [Starlite organization](https://github.com/starlite-api), based on the
+A Sphinx theme for the [Litestar organization](https://github.com/starlite-api), based on the
```

Comment:
Thanks for this, can you update the github URL here also please?

Reply:
This one seems to be correct. It doesn't redirect to anything.

Reply:
Nevermind. Got it.